bzira.py

```diff
@@ -187,11 +188,14 @@ def parse_options():
     
 bz = bugzilla.Bugzilla(url=bzserver + "bugs/xmlrpc.cgi")
 
-print "[DEBUG] " + "Querying bugzilla: " + query
+
+if (options.verbose):
+    print "[DEBUG] " + "Querying bugzilla: " + query.replace("last_change_time","chfieldfrom")
```

Comment:
https://bugs.eclipse.org/bugs/buglist.cgi?last_change_time=2016-01-27%2010%3A00&list_id=13714458&status_whiteboard=RHT ... 61 bugs found
https://bugs.eclipse.org/bugs/buglist.cgi?chfieldfrom=2016-01-27%2010%3A00&list_id=13714457&status_whiteboard=RHT ... 1 bug found.

Which one is actually using the date filter? The second one. Hence my fix.

If you're going to echo a debug statement to the console, it should be one that actually works.